src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1,2 @@
+com.amazonaws.services.codebuild.model.BuildPhase
+com.amazonaws.services.codebuild.model.PhaseContext
```

Comment:
I assume you checked these and they look safe?

Reply:
yes